### DISCUSSION TITLE
Add attribute texture upload to api webservice

### DISCUSSION THREAD

**Discussion by bilelbelghith** [PrestaShop/PrestaShop#31494] [Ideas]
### The problem

When using APIs to manage prestashop, there is no feature that allows me to upload attribute image texture via API, which requires the client to manually upload it from prestashop admin dashboard

### The solution

Allow developer to upload attribute texture via API.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [x] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hello @bilelbelghith 

That's a very good idea. It will be great to see a pull request from you!
